### ISSUE TITLE
ScreenReader is not announcing the state as 'Passed' or 'Failed' under query history section: A11y_mssql for VSCode_mssql Extension_Query History_Info and Relationships

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17882]
## Actual:
NVDA/JAWS is not announcing the state as 'Passed' or 'Failed' under query history section.

## Expected:
NVDA/JAWS should announce the state as 'Passed' or 'Failed' under query history section.

## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG1.3.1;#A11yMAS;#Win32;#InfoAndRelationships;#ScreenReader;#JAWS;#NVDA;#FTP;#STP;#Element:List;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code insider: 1.86.2
Extension Version: v1.22.1
ScreenReader: JAWS(2023.2212.23), NVDA(2023.1.3)

## Repro Steps:

1. Open Visual studio code
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server button.
4. Click on Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Click on “New Query” write any valid query and click on "Execute query button".
7. In the left panel Query History window will appears
8. Verify whether Screen reader is announcing the state as passed/failed or not.

## User Impact:
Screenreader user will not be able to get the information about the state if screen reader does not announce it

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![ppt2CB pptm  -  AutoRecovered](https://github.com/microsoft/vscode-mssql/assets/100269580/a9b4e357-a8d5-464b-9d1a-f7d221c88fb3)
![ppt2CB pptm  -  AutoRecovered1](https://github.com/microsoft/vscode-mssql/assets/100269580/b2e8d474-e228-4b11-82be-a4715f00d2f0)

**Comment by aniketr99** [comment]
#Regressed:08-12-24;#closed;

Regressed the bug in the latest environment
Issue is fixed. So, closing the bug. Attachment is added for the reference.
![Screenshot (551)](https://github.com/user-attachments/assets/2d52559f-6bb9-4996-909e-c44b688f44eb)
![Screenshot (550)](https://github.com/user-attachments/assets/881d597d-401a-408a-983b-5a81268233db)
